tensorflow/core/kernels/BUILD

```diff
@@ -606,6 +606,7 @@ cc_library(
         ":batch_space_ops",
         ":bcast_ops",
         ":bitcast_op",
+        ":broadcast_to_op",
```

Comment:
Why is this here?

Reply:
@alextp The dependency is added so that it is part of the `gen_array_ops.py`.